### PR TITLE
Adds support for el8 platforms

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

File: main.tf [hunk 1]
```diff
@@ -134,6 +134,24 @@ locals {
...
+      platform   = local.platform_info.lx
+    }
+
+    rhel8 = {
+      ami_regex  = "spel-minimal-rhel-8-hvm-\\d{4}\\.\\d{2}\\.\\d{1}\\.x86_64-gp2"
+      ami_search = "spel-minimal-rhel-8-hvm-*.x86_64-gp2"
+      platform   = local.platform_info.lx
+    }
+
+    centos8stream = {
```
Comment (eemperor) [inline main.tf:149]: Very nit but can you sort alphabetically like EL7?  😀

Comment (lorengordon) [reply main.tf]: sure done

**Review by eemperor — APPROVED** [review]
🚢

**Pull request merged**